python/main.py

```diff
@@ -98,8 +134,13 @@ async def get_image(image_name):
 class Item(BaseModel):
     name: str
     category: str
+    image_name: str = None
 
 
 def insert_item(item: Item):
-    # STEP 4-1: add an implementation to store an item
-    pass
+    with open(items, "r") as f:
+        data = json.load(f)
+
+    data["items"].append({"name": item.name, "category": item.category, "image": item.image_name})
```

Comment:
[nits]
pydanticの機能を活用すれば、以下のようによりシンプルに書けます。
```suggestion
    data["items"].append(item.model_dump())
```

Reply:
なるほど、書き換えます！